.github/actions/docker-build-and-push/action.yaml

```
@@ -181,8 +181,34 @@ runs:
         flavor: |
           latest=false
 
+    - name: Docker meta for autoware:visualizer
+      id: meta-visualizer
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.target-image }}
+        tags: |
+          type=raw,value=visualizer-${{ inputs.platform }}
+          type=raw,value=visualizer-${{ steps.date.outputs.date }}-${{ inputs.platform }}
+          type=ref,event=tag,prefix=visualizer-,suffix=-${{ inputs.platform }}
+        bake-target: docker-metadata-action-visualizer
+        flavor: |
+          latest=false
+
+    - name: Docker meta for autoware:simulator
+      id: meta-simulator
+      uses: docker/metadata-action@v5
+      with:
+        images: ghcr.io/${{ github.repository_owner }}/${{ inputs.target-image }}
+        tags: |
+          type=raw,value=simulator-${{ inputs.platform }}
+          type=raw,value=simulator-${{ steps.date.outputs.date }}-${{ inputs.platform }}
+          type=ref,event=tag,prefix=simulator-,suffix=-${{ inputs.platform }}
+        bake-target: docker-metadata-action-simulator
+        flavor: |
+          latest=false
+
     - name: Docker meta for autoware:universe-devel
-      id: meta-universe-devel
+      id: meta-devel
```

I think it should be reverted.

```suggestion
      id: meta-universe-devel
```

---

Yes, thanks